src/Converter/Standard.php

```diff
@@ -50,10 +50,23 @@ class Standard implements ConverterInterface
 
     protected $postfix = '';
 
+    protected $leftDelimiter = '<';
+
+    protected $rightDelimiter = '>';
+
+
     public function __construct(DOMNode $node, ConverterInterface $parentConverter = null)
     {
         $this->node = $node;
         $this->parentConverter = $parentConverter;
+
+        // use the delimiters from the parent, if one is present
+        //
+        if($this->parentConverter) {
```

Comment:
Isn't that a choice that should be done when *invoking* the constructor? 

I think adding the delimiters as optional parameters would ease the interface a bit. What you put in will be set. As opposed to setting stuff based on some funky logic that people might not expect